src/components/MenuItem.tsx

```diff
@@ -776,8 +777,8 @@ function MenuItem(
                                                         {shouldShowSubscriptRightAvatar ? (
                                                             <SubscriptAvatar
                                                                 backgroundColor={isHovered ? theme.activeComponentBG : theme.componentBG}
-                                                                mainAvatar={floatRightAvatars[0]}
-                                                                secondaryAvatar={floatRightAvatars[1]}
+                                                                mainAvatar={floatRightAvatars.at(0) ?? getDefaultIcon}
```

Comment:
```suggestion
                                                                mainAvatar={floatRightAvatars.at(0) ?? fallbackIcon}
```

Reply:
The types are incompatible if I change it like above, `fallbackIcon` is just a field in type `Icon`, we also need to have `source` and `type` and function `getDefaultIcon` provides these fields

<img width="775" alt="Screenshot 2024-09-24 at 13 11 01" src="https://github.com/user-attachments/assets/7231d5f4-954a-4e12-b2c1-efefc559a1a3">